pkg/controller/cainjector/injectors.go

```diff
@@ -17,10 +17,10 @@ limitations under the License.
 package cainjector
 
 import (
-	admissionreg "k8s.io/api/admissionregistration/v1beta1"
-	apiext "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	admissionreg "k8s.io/api/admissionregistration/v1"
+	apiext "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
 	"k8s.io/apimachinery/pkg/runtime"
-	apireg "k8s.io/kube-aggregator/pkg/apis/apiregistration/v1beta1"
+	apireg "k8s.io/kube-aggregator/pkg/apis/apiregistration/v1"
```

Comment:
* admissionregistration GA since K8S 1.16: https://github.com/kubernetes/kubernetes/commit/3dd8add5d25d29358fb239ac23461643f1d8effc
* apiextensions GA since K8S 1.16: https://github.com/kubernetes/kubernetes/commit/ef397e5b368d530e712165a4bb78ec482d739c2b
* apiregistration.k8s.io (aggregation) GA since K8S 1.10: https://github.com/kubernetes/kubernetes/commit/35ed5338b104f41c91efb2b842e92fded6d0b509

Reply:
Thanks for all the links! 😄